django_sorcery/db/models.py

```diff
@@ -423,3 +424,29 @@ def full_clean_flush_handler(session, **kwargs):
     for i in session.dirty | session.new:
         if isinstance(i, Base):
             i.full_clean()
+
+
+def force_auto_coerce(cls=None):
+
+    m = cls.__mapper__ if cls else sa.orm.mapper
+
+    from django_sorcery.field_mapping import get_field_mapper
+
+    @sa.event.listens_for(m, "mapper_configured")
+    def hook_coercers(mapper, class_):
+        field_mapper = get_field_mapper()(model=class_)
+        info = model_info(class_)
+
+        def _coerce(target, value, oldvalue, initiator, form_field):
+            return form_field.clean(value)
```

Comment:
maybe wrap error a bit better to include which field the error is for?

```python
try:
    return form_field.clean(value)
except ValidationError as e:
    raise ValidationError({name: e})
```

will need to pass prop.name below to `partial` though